tests/features/chadmin_zookeeper.feature

```diff
@@ -88,3 +88,42 @@ Feature: chadmin zookeeper commands.
     And we delete zookeepers nodes /test/a,/test/a/b on zookeeper01
     And we delete zookeepers nodes /test/c/d,/test/c on zookeeper01
     Then the list of children on zookeeper01 for zk node /test are empty
+
+
+  Scenario: Set finished to ddl with removed host.
+    # Create the fake cluster with removed host. 
+    When we execute command on clickhouse02
+    """
+    echo -e "
```

Comment:
Let's write a [step](https://github.com/yandex/ch-tools/blob/0ba8de49db877e2f087ff7ddec69f2c4fe9fbcd7/tests/steps/clickhouse.py) for this action, e.g. `When we put the  clickhouse config to path `/a/b/c.xml` with/without restarting`. And leave here only config content and hide shell specific things in the step.